### PR TITLE
[bn] Localizes `content/bn/docs/tutorials/kubernetes-basics/_index.html`

### PR DESCRIPTION
**Pull request by Acesif**
Signed-off-by: Asif Zubayer Palak asifpalak@gmail.com
Localizes `content/bn/docs/tutorials/kubernetes-basics/_index.html`  to Bengali
This is an initial PR as may need some improvements!
Tracking issue: #31677

### CODE REVIEW

**Comment by mitul3737** [comment]
Kindly remove `content/bn/docs/tasks/tools/install-kubectl-linux.md` file from this branch

**Comment by Acesif** [comment]
@mitul3737 Removed the file you mentioned

File: content/bn/docs/tutorials/kubernetes-basics/_index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+weight: 10
+card:
+  name: টিউটোরিয়াল
+  weight: 20
+  title: টিউটোরিয়াল
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/_index.html:14]:
```suggestion
<html lang="bn">
```

File: content/bn/docs/tutorials/kubernetes-basics/_index.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
+---
+title: কুবারনেটিস বেসিক শিখুন
+linkTitle: কুবারনেটিস বেসিক শিখুন 
+no_list: true
+weight: 10
+card:
+  name: টিউটোরিয়াল
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/_index.html:7]:
```suggestion
  name: tutorials
```

File: content/bn/docs/tutorials/kubernetes-basics/_index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
+---
+title: কুবারনেটিস বেসিক শিখুন
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/_index.html:2]:
```suggestion
title: কুবারনেটিসের বেসিক শিখুন
```

File: content/bn/docs/tutorials/kubernetes-basics/_index.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+linkTitle: কুবারনেটিস বেসিক শিখুন
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/_index.html:3]:
```suggestion
linkTitle: কুবারনেটিসের বেসিক শিখুন
```

File: content/bn/docs/tutorials/kubernetes-basics/_index.html [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+  <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+  <main class="content">
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>কুবারনেটিস বেসিক</h2> 
+        <p>এই টিউটোরিয়ালটি কুবারনেটস ক্লাস্টার অর্কেস্ট্রেশন সিস্টেমের মূল বিষয়গুলির একটি ওয়াকথ্রু প্রদান করে। প্রতিটি মডিউলে কুবারনেটসের প্রধান বৈশিষ্ট্য এবং ধারণাগুলির কিছু পটভূমি তথ্য রয়েছে এবং একটি ইন্টারেক্টিভ অনলাইন টিউটোরিয়াল অন্তর্ভুক্ত রয়েছে। এই ইন্টারেক্টিভ টিউটোরিয়ালগুলি আপনাকে নিজের জন্য একটি সাধারণ ক্লাস্টার এবং এর ধারকযুক্ত অ্যাপ্লিকেশনগুলি পরিচালনা করতে দেয়৷</p>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/_index.html:27]:
```suggestion
        <p>এই টিউটোরিয়ালটি কুবারনেটিস ক্লাস্টার অর্কেস্ট্রেশন সিস্টেমের মূল বিষয়গুলির একটি ধারণা প্রদান করে। প্রতিটি মডিউলে কুবারনেটিসের প্রধান বৈশিষ্ট্য এবং ধারণাগুলির কিছু পটভূমি তথ্য রয়েছে এবং একটি ইন্টারেক্টিভ অনলাইন টিউটোরিয়াল অন্তর্ভুক্ত রয়েছে। এই ইন্টারেক্টিভ টিউটোরিয়ালগুলি আপনাকে নিজের জন্য একটি সাধারণ ক্লাস্টার এবং এর ধারকযুক্ত অ্যাপ্লিকেশনগুলি পরিচালনা করতে দেয় ৷</p>
```

File: content/bn/docs/tutorials/kubernetes-basics/_index.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+      <div class="col-md-9">
+        <h2>কুবারনেটিস আপনার জন্য কী করতে পারে?</h2> 
+        <p>আধুনিক ওয়েব পরিষেবাগুলির সাথে, ব্যবহারকারীরা আশা করে যে অ্যাপ্লিকেশনগুলি ২৪/৭ উপলব্ধ থাকবে এবং বিকাশকারীরা সেই অ্যাপ্লিকেশনগুলির নতুন সংস্করণগুলি দিনে কয়েকবার স্থাপন করার আশা করে৷ কনটেইনারাইজেশন প্যাকেজ সফ্টওয়্যারকে এই লক্ষ্যগুলি পূরণ করতে সহায়তা করে, অ্যাপ্লিকেশনগুলিকে মুক্তি এবং ডাউনটাইম ছাড়াই আপডেট করতে সক্ষম করে। কুবারনেটিস আপনাকে নিশ্চিত করতে সাহায্য করে যে সেই কন্টেইনারাইজড অ্যাপ্লিকেশানগুলি আপনি যেখানে এবং যখন চান চালান, এবং তাদের কাজ করার জন্য প্রয়োজনীয় সংস্থান এবং সরঞ্জামগুলি খুঁজে পেতে সহায়তা করে৷ কুবারনেটিস হল একটি উৎপাদন-প্রস্তুত, ওপেন সোর্স প্ল্যাটফর্ম যা কনটেইনার অর্কেস্ট্রেশনে গুগল-এর সঞ্চিত অভিজ্ঞতার সাহায্যে ডিজাইন করা হয়েছে, যা সম্প্রদায়ের সেরা জাত ধারণাগুলির সাথে মিলিত হয়েছে</p>
+      </div>
+    </div>
+
+    <br>
+    
+    <div id="basics-modules" class="content__modules">
+      <h2>কুবারনেটিস বেসিক মডিউল</h2>
```
Comment (mitul3737) [inline content/bn/docs/tutorials/kubernetes-basics/_index.html:51]:
```suggestion
      <h2>কুবারনেটিসের বেসিক মডিউল</h2> 
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Review by mitul3737 — APPROVED** [review]
/lgtm

**Pull request merged**